_targets.R

```diff
@@ -46,29 +47,39 @@ metrics_med_DA <- c('mh15', 'mh16', 'mh17', 'mh21', 'mh24', 'mh27')
 #non-exceedance quantiles for additional metrics - daily flows
 NE_quants = c(seq(0.5, 0.95, 0.05), 0.98, 0.99, 0.995)
 
+
+###moving window parameters
+window_length <- 20  ##needs to be <= complete_years
+increment <- 1
```

Comment:
Did you test using other increments? In targets, I think we could `map` over a vector of increments to make plots for several increments with one pipeline build

Reply:
It might be `cross` instead of `map` as the pattern because I see you're mapping over sites already

Reply:
I did it with a 2 year increment just to make sure it worked ok but the 1 year increment didn't take an exceedingly long time, so I just left it at that. Lets talk about if this is necessary.  A plot with a longer increment will be the same as a 1-yr plot except with fewer points on it, so I'm not sure what that would tell us.

Reply:
Okay, thanks, that makes sense. We can leave it as 1-yr then